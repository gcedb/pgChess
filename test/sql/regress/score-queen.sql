\set my_game 'k7/8/8/2p5/8/8/2Q5/7K w - - 0 2'
\set pg2podg_iterations 1
\set pg2podg_depth_target 1
