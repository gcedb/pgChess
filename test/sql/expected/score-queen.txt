psql:test/sql/regress/_.sql:51: INFO:  1 turns to go (1 games)
ID: k7/8/8/2p5/8/8/2Q5/7K w - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  Score: -11.600
Choices:
	1.700 {1.700} k7/8/8/2Q5/8/8/8/7K b - - 0 2
	0.800 {0.800} k7/8/8/2p5/4Q3/8/8/7K b - - 0 2
	0.500 {0.500} k7/8/8/2p5/8/1Q6/8/7K b - - 0 2
	0.500 {0.500} k7/8/8/2p5/8/3Q4/8/7K b - - 0 2
	0.500 {0.500} k7/8/8/2p5/8/8/1Q6/7K b - - 0 2
	0.300 {0.300} k7/8/6Q1/2p5/8/8/8/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p2Q2/8/8/8/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p5/2Q5/8/8/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p5/8/8/2Q3K1/8 b - - 0 2
	0.300 {0.300} k7/8/8/2p5/8/8/3Q4/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p5/8/8/4Q3/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p5/8/8/6Q1/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p5/8/8/Q7/7K b - - 0 2
	0.300 {0.300} k7/8/8/2p5/Q7/8/8/7K b - - 0 2
	0.200 {0.200} k7/7Q/8/2p5/8/8/8/7K b - - 0 2
	0.200 {0.200} k7/8/8/2p5/8/2Q5/8/7K b - - 0 2
	0.200 {0.200} k7/8/8/2p5/8/8/2Q5/6K1 b - - 0 2
	0.200 {0.200} k7/8/8/2p5/8/8/8/1Q5K b - - 0 2
	0.100 {0.100} k7/8/8/2p5/8/8/2Q4K/8 b - - 0 2
	0.100 {0.100} k7/8/8/2p5/8/8/5Q2/7K b - - 0 2
	0.000 {0.000} k7/8/8/2p5/8/8/7Q/7K b - - 0 2
	0.000 {0.000} k7/8/8/2p5/8/8/8/3Q3K b - - 0 2
	-0.300 {-0.300} k7/8/8/2p5/8/8/8/2Q4K b - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  0 turns to go (24 games)
ID: k7/8/8/2Q5/8/8/8/7K b - - 0 2
psql:test/sql/regress/_.sql:51: INFO:  game ended after 1 iterations
